Representable/representable
# Removes tag model

Removes tag model and artifacts on form page.

Key steps to test:
- See that there are no errors after running migrations.
- Run server and check map and entry pages for weirdness. I was able to submit an entry with no issues but we'll need to double check
